views/director-nationality.html

```diff
@@ -150,20 +150,19 @@ <h2 class="govuk-heading-m">{{ publicRegisterInformation }}</h2>
       }
 
       var nationality2Error = document.querySelector('a[href="#typeahead_input_1"]')
-      if (nationality2Error) {
```

Comment:
Do these changes not duplicate the logic from 115 above? They should expose the links / form groups as required if the nunjucks parameters are provided.

Reply:
@jbestwickch removed and reverted to existing logic with parameters parsed